web/src/inbox_ui.js

```diff
@@ -653,6 +653,28 @@ function revive_current_focus() {
     }
 }
 
+function update_closed_compose_text($row, not_a_header_row) {
+    if (!not_a_header_row) {
```

Comment:
Let's add a copy of the "TODO" comment here too.

Reply:
Done.